src/main/java/com/yahoo/identity/services/storage/sql/SqlCredentialService.java

```diff
@@ -0,0 +1,15 @@
+package com.yahoo.identity.services.storage.sql;
+
+import com.yahoo.identity.services.credential.Credential;
+import com.yahoo.identity.services.credential.CredentialService;
+
+import javax.annotation.Nonnull;
+
+public class SqlCredentialService implements CredentialService {
```

Comment:
Why the implementation is called SqlCredentialService? It shouldn't need to talk to database.

Reply:
> Why the implementation is called SqlCredentialService? It shouldn't need to talk to database.

The current design doesn’t have that file. It is the redundant file brought when rebasing with historical commit. Already deleted.

Reply:
The current design doesn’t have that file. It is the redundant file brought when rebasing with historical commit. Already deleted.